templates/vertical-interactive-map/page.html.hbs

```diff
@@ -17,10 +17,10 @@
     {{> templates/vertical-interactive-map/script/locationbias modifier="main" }}
     {{> templates/vertical-interactive-map/script/locationbias modifier="mobileMap" }}
   {{/script/core }}
-  <script src="{{relativePath}}/locator-components.js" defer></script>
+  <script src="{{relativePath}}/locator-bundle.js" defer></script>
```

Comment:
I didn't know this bundle existed, would it make more sense to put the card listener class in here instead, if it's only meant to be used on the locator page?

Reply:
I did put the card listener class in that bundle (rather than in HitchhikerJS)

Reply:
sorry missed that